enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1960,36 +2035,36 @@ func (ws *workspace) fetch(ctx context.Context, remoteURL string, refs []string,
 	return nil
 }
 
-type gitError struct {
+type commandErr struct {
```

Comment:
nit: use unabbreviated name since it's a type

```suggestion
type commandError struct {
```